src/debugger/Debugger.js

```diff
@@ -60,7 +62,6 @@ export class DebugServer {
   // Checking if the debugger needs to take any action on reaching this ast node
   checkForActions(ast: BabelNode) {
     this.checkForBreakpoint(ast);
```

Comment:
I do not think we should special check for breakpoint here. What you want to do here is "processDebuggerCommand()" here, not necessary breakpoints, there may be other commands you want to process from UI, like async break.

Reply:
`checkForActions` is the entry point into the debugger from Prepack. On each AST node that is evaluated, this method is called to check if the debugger needs to perform any actions on this AST node. Reading in commands from the UI is done in `waitForRun`

Reply:
I see. The github collapse code confused me during review.